tests/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM lsstsqre/centos:7-stackbase-devtoolset-8
```

Comment:
This Dockerfile does not appear to do anything useful in terms of testing lsstinstall.  Get rid of it and the Makefile.